Tests/otlLib/builder_test.py

```diff
@@ -1402,6 +1402,85 @@ def test_buildStatTable(axes, axisValues, elidedFallbackName, expected_ttx):
     assert expected_ttx == ttx
 
 
+def test_buildStatTable_name_duplicates():
```

Comment:
Does the main premise of this test need updating after your latest round of changes?

Perhaps you can minimize it somewhat? Or try to focus it more on the problem that you're fixing.

Reply:
Also: I would replace the doc string with a comment. Perhaps slightly reworded to be relevant to the "fixed state", instead of saying "currently" while meaning the broken past :)

Reply:
I think you could rename `test_buildStatTable_name_duplicates` to `test_buildStatTable_platform_specific_names`; the main change in this PR is the addition of `windowsNames` and `macNames` parameter to `buildStatTable`, not the behavior of `addMultilingualNames` as regards name duplicates, which hasn't changed and was already used by `buildStatTable`.